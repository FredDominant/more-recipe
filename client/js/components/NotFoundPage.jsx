import React from 'react';
import Navbar from '../components/Navbar';
import Footer from './Footer';

const NotFoundPage = () => (
  <div>
    <Navbar />
    <div className="container">
      <br />
      <h1 className="text-center">
        <strong> 404 </strong>
      </h1>
      <br />
      <h3 className="text-center">
        The resource you are looking for does not exist, or has been taken off the catalogue
      </h3>
    </div>
    <Footer />
  </div>
);
export default NotFoundPage;
