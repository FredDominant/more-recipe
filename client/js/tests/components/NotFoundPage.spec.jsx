import React from 'react';
import { shallow } from 'enzyme';

import NotFoundPage from '../../components/NotFoundPage';
/**
 * @description test for Body component
 *
 * @returns {null} null
 */
function setup() {
  const wrapper = shallow(<NotFoundPage />);

  return {
    wrapper
  };
}

describe('Test for NotFoundPage component', () => {
  it('should render correctly', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('strong').length).toBe(1);
    expect(wrapper.find('div').length).toBe(2);
    expect(wrapper.find('.text-center').length).toBe(2);
  });
});
