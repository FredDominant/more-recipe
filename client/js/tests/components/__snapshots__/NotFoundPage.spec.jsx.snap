// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for NotFoundPage component should render correctly 1`] = `
<div>
  <div
    className="container notFound-body"
  >
    <br />
    <h1
      className="text-center"
    >
      <strong>
         404 
      </strong>
    </h1>
    <br />
    <h3
      className="text-center mb-10"
    >
      The resource you are looking for does not exist, or has been taken off the catalogue
    </h3>
  </div>
  <br />
  <Footer />
</div>
`;
